docs/appendices/release-notes/5.5.3.rst

```diff
@@ -112,3 +112,6 @@ Fixes
         modification_date TIMESTAMP AS current_timestamp
     )
     INSERT INTO tbl (id, a) VALUES (1, 2) ON CONFLICT (id) DO UPDATE SET a = 3;
+
+- Fixed a performance regression introduced in 5.5.0 which prevented caching of
+  predicates like ``abs(x) = 1`` which require a scalar function evaluation.
```

Comment:
that's not true, will remove it. it's other way around, it's currently always Cacheable.